library/nginx/1.15/config.yaml

```diff
@@ -0,0 +1,2 @@
+networking: True
+memory: 8
```

Comment:
Isn't 8mb of memory way too less?

Reply:
No. Test it and you'll see. The beauty of native apps.